e2eRunner/src/main/scala/za/co/absa/hermes/e2eRunner/plugins/BashPlugin.scala

```diff
@@ -53,13 +53,17 @@ class BashPlugin extends Plugin {
 
   override def performAction(testDefinition: TestDefinition, actualOrder: Int): BashJobsResult = {
     def runBashCmd(bashCmd: String): String = {
-      (s"echo $bashCmd" #| "bash").!!
+      if (System.getProperty("os.name").toLowerCase().contains("windows")) {
+        (s"cmd /C ${bashCmd.replace("'", "\"")}").!!
```

Comment:
**Minor:** would rename `bashCmd` to just `cmd`. And maybe the `runBashCmd` to `runCmd` or `runOSCmd`

Reply:
Renamed in commit 5fa69b3df0ffbe899bfd6725207517b0daf85abd.